Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -938,16 +938,14 @@ bool MMU::IsOptimizableRAMAddress(const u32 address, const u32 access_size) cons
 }
 
 template <XCheckTLBFlag flag>
-bool MMU::IsRAMAddress(u32 address, bool translate)
+bool MMU::IsEffectiveRAMAddress(u32 address)
 {
-  if (translate)
-  {
-    auto translate_address = TranslateAddress<flag>(address);
-    if (!translate_address.Success())
-      return false;
-    address = translate_address.address;
-  }
+  auto translate_address = TranslateAddress<flag>(address);
```

Comment:
Can this variable be const as mitaclaw suggested?
```suggestion
  const auto translate_address = TranslateAddress<flag>(address);
```